src/golang/lib/job/spark.go

```diff
@@ -27,7 +27,7 @@ func NewSparkJobManager(conf *SparkJobManagerConfig) (*SparkJobManager, error) {
 
 	session, err := livyClient.CreateSession(&spark.CreateSessionRequest{
 		Kind:                     "pyspark",
-		HeartbeatTimeoutInSecond: 600,
+		HeartbeatTimeoutInSecond: 10,
```

Comment:
I assume this was a necessary change to get it working with the test suite? This seems like quite a large numeric change - what are the ramifications of this?